src/ocean/text/convert/Formatter_test.d

```diff
@@ -423,10 +429,34 @@ unittest
         FooBar = 42
     }
 
-    Foo f = Foo.FooBar;
-    test("42" == format("{}", f));
-    f = cast(Foo)36;
-    test("36" == format("{}", f));
+    string[] outputs;
+    outputs.length = 3;
+    assumeSafeAppend(outputs);
+
+    Foo[] foo_inputs;
+    foo_inputs.length = 3;
+    assumeSafeAppend(foo_inputs);
+    testNoAlloc(foo_inputs.fill(Foo.FooBar, cast(Foo)1, cast(Foo)36));
+    testNoAlloc(outputs.fill("Foo.FooBar", "Foo.B", "cast(Foo) 36"));
+    foreach (i, ref item; foo_inputs)
+        test!("==")(outputs[i], format("{}", item));
```

Comment:
```suggestion
    char[256] buffer;
    immutable Foo[] foo_inputs = [ Foo.FooBar, cast(Foo)1, cast(Foo)36 ];
    immutable string[] foo_outputs = [ "Foo.FooBar", "Foo.B", "cast(Foo) 36" ];
    foreach (i, ref item; foo_inputs)
        testNoAlloc(assert(sformat(buff, "{}", item) == output[i]));
```

We want to test that `sformat` does not allocate. The test currently test that `fill` does not allocate. Why ?

Reply:
> We want to test that `sformat` does not allocate. The test currently test that `fill` does not allocate. Why ?

I was mistaken.

Reply:
sformat(buffer, "{}", item)   =>
result is "immutable(Foo).FooBar"